### PR TITLE
feat: compatible wallets page

### PR DESCRIPTION
**Pull request by ItsANameToo**
## Summary

https://app.clickup.com/t/861mru9n1

TODO:
- [x] navy theme colors
- [x] some dark mode adjustments
- [x] awaiting dark mode image

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: config/blade-icons.php [hunk 1]
```diff
@@ -71,6 +71,12 @@
             'class'  => 'fill-current',
         ],
 
+        'wallets' => [
+            'path'   => 'resources/icons/wallets',
```
Comment (alexbarnsley) [inline config/blade-icons.php:75]: Could just do `<x-ark-icon name="app-wallets.exodus">` I think :thinking:

File: resources/views/app/compatible-wallets.blade.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                <div class="flex flex-1 py-2 pr-3">
+                    <img src="{{ mix('images/wallets/arkvault.svg') }}" class="dark:hidden" />
+                    <img src="{{ mix('images/wallets/arkvault-dark.svg') }}" class="hidden dark:block" />
+                </div>
+            </div>
+
+            <div class="grid grid-cols-1 gap-3 mt-6 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5">
+                @foreach (trans('pages.compatible-wallets.wallets') as $wallet)
+                    <div class="flex flex-col bg-white rounded-xl border border-theme-secondary-300 dark:bg-theme-secondary-900 dark:border-theme-secondary-800">
+                        <div class="flex justify-center items-center mx-2 mt-2 rounded-xl aspect-square"> <x-ark-icon name="app.wallets-{{$wallet['logo']}}" size="none" /></div>
```
Comment (alexbarnsley) [inline resources/views/app/compatible-wallets.blade.php:43]:
```suggestion
                        <div class="flex justify-center items-center mx-2 mt-2 rounded-xl aspect-square">
                            <x-ark-icon name="app.wallets-{{$wallet['logo']}}" size="none" />
                        </div>
```

File: resources/views/app/compatible-wallets.blade.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                        <div class="mx-6 mt-3 mb-6">
+                            <x-ark-external-link :url="$wallet['url']">{{ $wallet['title'] }}</x-ark-external-link>
+                        </div>
+                    </div>
+                @endforeach
+            </div>
+
+            <div class="text-center sm:text-start flex-col sm:flex-row mt-6 w-full flex py-6 sm:py-2 px-6 justify-between items-center dark:from-theme-secondary-800 dark:via-theme-secondary-800 dark:to-theme-secondary-800 bg-gradient-to-r from-[#E5F0F8] via-[#FFF8EB] to-[#E5F0F8] rounded-xl">
+                <span class="text-lg font-semibold dark:text-white text-theme-primary-900">@lang('pages.compatible-wallets.get_listed')</span>
+                <button type="button" class="mt-3 w-full sm:mt-0 sm:w-auto button-primary">@lang('actions.submit_wallet')</button>
```
Comment (alexbarnsley) [inline resources/views/app/compatible-wallets.blade.php:53]:
```suggestion
                <span class="text-lg font-semibold dark:text-white text-theme-primary-900">
                    @lang('pages.compatible-wallets.get_listed')
                </span>
                <button type="button" class="mt-3 w-full sm:mt-0 sm:w-auto button-primary">
                    @lang('actions.submit_wallet')
                </button>
```

File: resources/views/app/compatible-wallets.blade.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            </div>
+
+            <div class="flex flex-col mt-6 w-full rounded-xl border min-[960px]:flex-row border-theme-secondary-300 dark:border-theme-secondary-800">
+                <div class="flex flex-col flex-1 justify-between py-8 px-8 space-y-6">
+                    <span class="text-sm items-center rounded font-semibold px-2 py-1 flex space-x-2 text-theme-secondary-900 dark:text-white dark:from-theme-navy-600 dark:via-theme-navy-600 dark:to-theme-navy-600 bg-gradient-to-r from-[#E5F0F8] via-[#FFF8EB] to-[#E5F0F8]">
+                        <x-ark-icon name="circle.info" />
+                        <span>@lang('pages.compatible-wallets.arkvault.disclaimer')</span>
+                    </span>
+                    <div>
+                        <h2 class="text-2xl font-semibold text-theme-secondary-900">@lang('general.arkvault') <span class="text-theme-secondary-500 dark:text-theme-secondary-700">(@lang('pages.compatible-wallets.arkvault.web_wallet'))</span></h2>
```
Comment (alexbarnsley) [inline resources/views/app/compatible-wallets.blade.php:18]:
```suggestion
                        <h2 class="text-2xl font-semibold text-theme-secondary-900">
                            @lang('general.arkvault') 
                            <span class="text-theme-secondary-500 dark:text-theme-secondary-700">
                                (@lang('pages.compatible-wallets.arkvault.web_wallet'))
                            </span>
                        </h2>
```

File: resources/views/app/compatible-wallets.blade.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                            <div><x-ark-icon name="app.wallets-arkvault" size="none" class="w-10 h-10 dark:text-white text-theme-navy-600" /></div>
+                            <div class="flex flex-col ml-3">
+                                <span class="text-lg font-semibold dark:text-white text-theme-secondary-900">@lang('general.arkvault')</span>
+                                <span class="text-sm font-semibold text-theme-secondary-700 dark:text-theme-dark-blue-400">@lang('pages.compatible-wallets.arkvault.subtitle')</span>
+                            </div>
+                        </div>
+                        <div class="flex items-center">
+                            <a href="@lang('pages.compatible-wallets.arkvault.url')" target="_blank" rel="noopener nofollow noreferrer" class="flex items-center mt-4 w-full h-full rounded-xl sm:mt-0 sm:w-auto md:mt-0 md:w-full lg:w-auto button-primary"><div class="flex justify-center items-center h-full"><span>@lang('actions.learn_more')</span></div></a>
```
Comment (alexbarnsley) [inline resources/views/app/compatible-wallets.blade.php:30]: Hard to read :sadmici: if it needs no new lines to handle annoying auto-spacing you could try `inline` or `inline-flex`

```suggestion
                            <a 
                                href="@lang('pages.compatible-wallets.arkvault.url')"
                                target="_blank"
                                rel="noopener nofollow noreferrer"
                                class="flex items-center mt-4 w-full h-full rounded-xl sm:mt-0 sm:w-auto md:mt-0 md:w-full lg:w-auto button-primary"
                            >
                                <div class="flex justify-center items-center h-full">
                                    <span>@lang('actions.learn_more')</span>
                                </div>
                            </a>
```

File: resources/views/app/compatible-wallets.blade.php (hunk 3 above)
Comment (alexbarnsley) [reply resources/views/app/compatible-wallets.blade.php:53]: you get the idea :trollface:

**Review by alexbarnsley — CHANGES_REQUESTED** [review]
Issues I've noticed:

There shouldn't be the bordered container on 320-767px
Design (320px):
image

Implementation (320):
image

Design (640)
image

Implementation (640)
image

It should have a separating border instead:
image


-----------------------

Hard to tell due to the above issue but it looks like the banner as too much padding on the right

Design (320)
image


Implementation (320)
image

----------

I originally tried fixing it myself but I left it for now as it was fiddly (could be a future task). The banner size is slightly different to design. It looks like the content on the left should be justify-centre'd with the correct spacing, instead of justify-between. Notice spacing above "ARK Desktop Wallet is discontinued" and below the CTA

Design
image

Implementation
image

------ 

We've already discussed the 12px font in the header :trollface: I guess it's worth checking with Oleg since the padding below the heading is 8px, versus the 6px in design

Design
image

Implementation
image

------ 

Navy colour isn't used in dark mode for links or buttons

Design
image

image

image

Impl
image

image

image

---------------

Text in bottom CTA should be 16px 

design
image

impl
image

File: config/blade-icons.php (hunk 1 above)
Comment (ItsANameToo) [reply config/blade-icons.php:75]: negative

Comment (ItsANameToo) [reply config/blade-icons.php:75]: actually it does, I forgot to update another reference

**Comment by ItsANameToo** [comment]
updated it @alexbarnsley , the design has been changed to reflect the regular button colors instead of the navy ones

**Review by alexbarnsley — CHANGES_REQUESTED** [review]
Looks good other than a weird gap I can't explain between the heading and the 12px text, and also the left padding in the below CTA is too large:

image

**Pull request merged**